metrics/src/main/java/com/lightstep/tracer/metrics/ProtobufSender.java

```diff
@@ -1,5 +1,7 @@
 package com.lightstep.tracer.metrics;
 
+import static com.lightstep.tracer.shared.LightStepConstants.Tags.*;
```

Comment:
Ugh, again, why is this changed even needed? You are not using any tag other than Component.